recipes/deploy-admin.rb

```diff
@@ -82,6 +82,7 @@
     install_multitenancy_config(most_recent_deploy, public_admin_hostname, public_engage_hostname)
     remove_felix_fileinstall(most_recent_deploy)
     install_smtp_config(most_recent_deploy)
+    install_default_tentant_config(most_recent_deploy, public_admin_hostname, private_hostname)
```

Comment:
Does it need to be active on the admin node? I'm guessing we'd only need it on nodes that run the engage-related services.

FWIW, I have no problem with configuring nodes identically even if the configs aren't used on that node type - I think it's a little clearer, actually. But if we're going to do that, then let's add this to `deploy-worker` too.